cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -0,0 +1,162 @@
+package com.powsybl.cgmes.conversion;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.powsybl.cgmes.conversion.RegulatingControlMapping.RegulatingControl;
+import com.powsybl.cgmes.model.CgmesModelException;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.StaticVarCompensator;
+import com.powsybl.iidm.network.StaticVarCompensatorAdder;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author José Antonio Marqués <marquesja at aia.es>
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+
+public class RegulatingControlMappingForStaticVarCompensators {
+
+    public RegulatingControlMappingForStaticVarCompensators(RegulatingControlMapping parent) {
+        this.parent = parent;
+        this.context = parent.context();
+        mapping = new HashMap<>();
+    }
+
+    public static void initialize(StaticVarCompensatorAdder adder) {
+        adder.setVoltageSetPoint(Double.NaN);
+        adder.setReactivePowerSetPoint(Double.NaN);
+        adder.setRegulationMode(StaticVarCompensator.RegulationMode.OFF);
```

Comment:
You can only set `regulationMode` here, `voltageSetPoint` and `reactivePowerSetPoint` already equal to `NaN` by default.

Reply:
Done